aiida/cmdline/commands/work.py

```diff
@@ -8,12 +8,19 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 import click
+import logging
 from tabulate import tabulate
 from aiida.cmdline.commands import work, verdi
 from aiida.cmdline.baseclass import VerdiCommandWithSubcommands
 
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
+LOG_LEVELS = [
```

Comment:
Ah, I think I was looking for that. I'm not sure if we should use ``_levelNames`` because it's not in the public API.. but then again I agree that it's nicer than the alternative. I just updated to filter "manually", which has the benefit of allowing arbitrary log levels in the report.